### PR TITLE
log TFLite profile in CSV format

### PR DESCRIPTION
**Pull request by danc86**
Restores the previous behaviour, which was accidentally lost in PR #157.

### CODE REVIEW

File: common/src/tflite.cc [hunk 1]
```diff
@@ -248,7 +248,7 @@ void tflite_classify() {
   }
   uint32_t end = perf_get_mcycle();
 #ifndef NPROFILE
-  profiler->Log();
+  profiler->LogCsv();
 #endif
   perf_print_all_counters();
```
Comment (alanvgreen) [inline common/src/tflite.cc:253]: Please also move perf_print_all_counters() inside the #if so performance counters are not printed if profiling has been turned offf.

Comment (danc86) [reply common/src/tflite.cc:253]: Ahh I see. They are always zero if NPROFILE is set because all the code which touches perf counters is compiled out.

**Comment by danc86** [comment]
Hmm seems like CFU-Playground has the setting to deny non-fast-forwards on PR merges. It might be worth changing that setting now that we are regularly merging several PRs per day. It tends to create pointless merge commits (in the best case). I personally like the "rebase if necessary" strategy which matches what Gerrit does.

**Comment by danc86** [comment]
I'll rebase this as soon as the other PR is landed.

**Comment by tcal-x** [comment]
> Hmm seems like CFU-Playground has the setting to deny non-fast-forwards on PR merges. It might be worth changing that setting now that we are regularly merging several PRs per day. It tends to create pointless merge commits (in the best case). I personally like the "rebase if necessary" strategy which matches what Gerrit does.

I'd be happy to change this.

**Review by alanvgreen — APPROVED** [review]
LGTM

**Pull request merged**